Add serverless deployment tracking documentation (#8939)
DataDog/documentation: content/en/serverless/deployment_tracking/_index.md

---
title: Deployment Tracking
kind: documentation
further_reading:
- link: "/serverless/distributed_tracing"
  tag: "Documentation"
  text: "Distributed Tracing for Serverless"
- link: "/serverless/serverless_tagging"
  tag: "Documentation"
  text: "Serverless Tagging"
---

{{< img src="serverless/deployment_tracking.jpeg" alt="Serverless Deployment Tracking"  style="width:100%;">}}

> Deployment Tracking can help you understand when a new version of code or a configuration change causes a spike in errors, degraded performance, or when your cloud environment drifts from its expected state.

Reviewer: ```suggestion
Deployment tracking helps you to understand when a new version of code or a configuration change causes a spike in errors, degraded performance, or when your cloud environment drifts from its expected state.
```

> To access Deployment Tracking for your serverless applications, select a function in the [Serverless view][1] and click the “Deployments” tab. Here you can see key serverless metrics like invocations, execution duration, and error counts automatically displayed with event overlays that mark code deployments and configuration changes related to the function.

Reviewer: ```suggestion
To access deployment tracking for your serverless applications, select a function in the [Serverless view][1] and click the **Deployments** tab. This pages shows key serverless metrics like invocations, execution duration, and error counts automatically displayed with event overlays that mark code deployments and configuration changes related to the function.
```

For visibility into historical code and configuration changes, adjust the global time selector at the top right of the view.

> ## Enabling Deployment Tracking

Reviewer: ```suggestion
## Setup
```

Datadog collects code and configuration change events for your AWS Lambda functions from AWS CloudTrail.

> If you haven’t already, set up the [Amazon Web Services][2] integration first. Then, add the following permission to the policy document for your AWS/Datadog Role:

Reviewer: ```suggestion
If you haven’t already, set up the [Amazon Web Services][2] integration first. Then, add the following permission to the policy document for your AWS/Datadog role:
```
